cmd/zetae2e/local/rpc.go

```diff
@@ -0,0 +1,59 @@
+package local
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fatih/color"
+
+	"github.com/zeta-chain/node/e2e/config"
+	"github.com/zeta-chain/node/e2e/e2etests"
+	"github.com/zeta-chain/node/e2e/runner"
+)
+
+// rpcTestRoutine runs zevm json rpc tests
+func rpcTestRoutine(
+	conf config.Config,
+	deployerRunner *runner.E2ERunner,
+	verbose bool,
+	testNames ...string,
+) func() error {
+	return func() (err error) {
+		account := conf.AdditionalAccounts.UserRPC
+		// initialize runner for rpc test
+		rpcTestRunner, err := initTestRunner(
+			"rpc",
+			conf,
+			deployerRunner,
+			account,
+			runner.NewLogger(verbose, color.FgMagenta, "rpc"),
+			runner.WithZetaTxServer(deployerRunner.ZetaTxServer),
+		)
+		if err != nil {
+			return err
+		}
+
+		rpcTestRunner.Logger.Print("🏃 starting RPC tests")
+		startTime := time.Now()
+
+		if len(testNames) == 0 {
```

Comment:
nit: the `rpcTestRunner.RunE2ETests` already handles empty list internally. Do we really need this special handling?